### ISSUE TITLE
fr.d2i: The process cannot access the file because it is being used by another process.

### ISSUE THREAD

**Issue by verdie-g** [dofusdude/doduda#7]
```
go install github.com/dofusdude/doduda@latest
doduda

fr ███████████████████████████████░░░░  87%

2023/09/18 19:13:27 FATA remove XXX\data\languages\fr.d2i: The process cannot access the file because it is being used by another process.
```

On Windows 11 x64

**Comment by stelzo** [comment]
I could not reproduce on Linux and Mac and I never tried it on Windows because I don't have any Windows PC to do it.
Do you have WSL and does it run there? I will install a Windows VM for this later.

**Comment by stelzo** [comment]
@verdie-g I managed to reproduce it with a virtualized Windows 11 x64. It is fixed with the linked PR and will be released right after with v0.2.4. Rerun your `go install` and it should work.

Also, I had some lag at the end after downloading everything on Windows only. Maybe it is just inside the VM? I will look at it some time later since it does not affect features. If it does not exits by itself at the end, just press `Enter`.
